<launch>
  <remap from="/joint_states" to="/robot/joint_states"/>
	<node name="select_grasps" pkg="grasp_selection" type="selection_node" output="screen">
  	<param name="cloud_topic" value="/register_clouds/point_cloud" />
    <param name="grasps_topic" value="/find_grasps/grasps" />
    <param name="joint_states_topic" value= "/joint_states" />
    <param name="marker_lifetime" value="60" />
    <param name="uses_scoring" value="true" />
    
		<!-- Reachibility Parameters -->
    <rosparam param="workspace"> [0.6, 1.0, -0.26, 0.14, -0.23, 1] </rosparam>
    <param name="min_aperture" value="0.02" />
    <param name="max_aperture" value="0.07" />
    <param name="num_additional_grasps" value="0" />
    <rosparam param="axis_order"> [2, 0, 1] </rosparam>
    <param name="planning_frame" value="/base" />
    <param name="hand_offset" value="0.095" />
    <param name="arm_link" value="right_gripper" />
    <param name="move_group" value="right_arm" />
    <param name="max_colliding_points" value="1" />
    <param name="JS_first_joint_index" value="9" />
    <param name="JS_last_joint_index" value="15" />
    <param name="IK_first_joint_index" value="8" />
    <param name="IK_last_joint_index" value="14" />
    <param name="prints" value="true" />
    
    <!-- Scoring Parameters -->
    <param name="urdf" value="/home/baxter/baxter_ws/src/baxter_common/baxter_description/urdf/baxter.urdf" />    
    <param name="num_selected" value="50" />
	</node>
</launch>
